src/meta/meta_bulk_load_service.cpp

```diff
@@ -1460,6 +1519,7 @@ void bulk_load_service::sync_apps_from_remote_storage()
         std::move(path), [this](bool flag, const std::vector<std::string> &children) {
             if (flag && children.size() > 0) {
                 ddebug_f("There are {} apps need to sync bulk load status", children.size());
+                dassert(children.size() == 1, "fatal, more than one app are processing bulkload");
```

Comment:
I suggest removing `dassert` check for more than one app is bulk load. In normal cases, it won't happen, but if it happens, meta server can not work totally for this assert. Meta server is important for system, concurrent bulk load process should not affert normal functions.
The same as the following assert in function `try_to_continue_bulk_load`, `do_continue_app_bulk_load`.